strings/ion_laws.json

```diff
@@ -1,5 +1,132 @@
 {
-    "ionabstract": [
+    "ionspecies": [
+        "LIFEFORMS WITHOUT TAILS",
+		"LIFEFORMS WITHOUT WINGS",
+		"INORGANIC LIFEFORMS",
+		"LIFEFORMS WITH TAILS",
+		"LIFEFORMS WITH WINGS",
+		"LIFEFORMS HOLDING A SOURCE OF LIGHT",
+		"LIFEFORMS HOLDING A TOOLBOX",
+		"LIFEFORMS HOLDING A FIRE EXTINGUISHER",
+		"LIFEFORMS WITH AN OBSCURED IDENTITY",
+		"LIFEFORMS THAT HAVE FARTED RECENTLY",
+		"LIFEFORMS THAT ARE NOT WEARING A BAG",
+		"LIFEFORMS THAT ARE WEARING A PAPER HAT",
+		"LIFEFORMS THAT ARE WEARING A CLOAK",
+		"LIFEFORMS THAT ARE NOT WEARING SHOES",
+		"LIFEFORMS THAT HAVE RECENTLY SAID NYA"
+    ],
+    "ionfragment": [
+        "VISIBLY CARRIES A GUN",
+		"HAS BLOOD ON THEIR CLOTHING",
+		"BREAKS SPACE LAW",
+		"HAS ANY DISEASE",
+		"IS WEARING A HARDSUIT"
+    ],
+    "iondepartment": [
+        "SECURITY",
+		"ENGINEERING",
+		"THE MEDBAY",
+		"RESEARCH AND DEVELOPMENT",
+		"MAINTENANCE",
+		"BOTANY",
+		"CARGO",
+		"SPACE"
+    ],
+    "ionkeyword": [
+        "VALID",
+		"GAMER",
+		"SEX",
+		"CUM",
+		"SHITSEC",
+		"ANTAG",
+		"SHUTTLE",
+		"CATGIRL",
+		"MALF",
+		"ANY CURSE WORD"
+    ],
+    "ionaction": [
+		"GETTING DRUNK",
+		"RUNNING",
+        "BEING HANDCUFFED",
+		"EATING",
+		"SURGERY",
+		"SQUEAKING OR HONKING"
```

Comment:
Suggestion: Potential to add a lot more here IE
 - Not Moving, doing their job, not doing their job, standing near some area, breaking into a department, damaging station equipment, using a spray can, working for an enemy corporation